server/views/partials/detail/aliases.njk

```diff
@@ -1,18 +1,22 @@
 {% from "../../macros/summaryCardMacro.njk" import summaryCard %}
 {% from "../../macros/summaryListRow.njk" import summaryListRow %}
+{% from "govuk/components/table/macro.njk" import govukTable %}
 
 {% if aliases %}
+  {% set aliasRows = [] %}
+  {% for alias in aliases %}
+    {% set attribute = { "data-qa": "alias" + loop.index0 } %}
+    {% set attributeDob = { "data-qa": "aliasDob" + loop.index0 } %}
+    {% set aliasRows = (aliasRows.push([
```

Comment:
I don't really understand the reassignment... I thought push returned the length of the array after the insert?

Reply:
Oh right push then return aliasRows. Presumably there's a reason you can't just do aliasRows.push()

Reply:
main reason is that I copied the code from [prisoner profile](https://github.com/ministryofjustice/hmpps-prisoner-profile/blob/dd028fc86ad9111b3689973d83f70ff257e3e34a/server/views/pages/locationDetails.njk#L104).  Note that this is nunjucks and not javascript - just found https://dev.to/giulia_chiola/add-items-to-an-array-in-nunjucks-482e